Beams/Testing/Cxx/vtkSlicerIECTransformLogicTest1.cxx

```diff
@@ -244,19 +337,381 @@ int vtkSlicerIECTransformLogicTest1(int vtkNotUsed(argc), char* vtkNotUsed(argv)
   if ( !IsTransformMatrixEqualTo(mrmlScene,
       iecLogic->GetTransformNodeBetween(vtkSlicerIECTransformLogic::Collimator, vtkSlicerIECTransformLogic::Gantry),
       expectedCollimatorToGantryTransform_90_MatrixElements ) )
-    {
+  {
     std::cerr << __LINE__ << ": CollimatorToGantry does not match baseline" << std::endl;
     return EXIT_FAILURE;
-    }
+  }
 
   double expectedBeamTransform_Collimator90_MatrixElements[16] =
     {  0, 0, -1, 1000,   0, 1, 0, 200,   1, 0, 0, 0,   0, 0, 0, 1  };
   if ( !IsTransformMatrixEqualTo(mrmlScene,
       beamTransformNode, expectedBeamTransform_Collimator90_MatrixElements ) )
-    {
+  {
     std::cerr << __LINE__ << ": Beam transform does not match baseline for collimator 90 degree angle" << std::endl;
     return EXIT_FAILURE;
-    }
+  }
+
+  // Patient support angle, "-1" degree
+  beamNode->SetCouchAngle(-1.0);
+  iecLogic->UpdateBeamTransform(beamNode);
+  expectedNumOfNonIdentityTransforms = 5;
+  if ((numOfNonIdentityTransforms = GetNumberOfNonIdentityIECTransforms(mrmlScene)) != expectedNumOfNonIdentityTransforms)
+  {
+    std::cerr << __LINE__ << ": Number of non-identity linear transforms: " << numOfNonIdentityTransforms << " does not match expected value: " << expectedNumOfNonIdentityTransforms << std::endl;
+    return EXIT_FAILURE;
+  }
+  double expectedPatientSupportRotationToFixedReferenceTransformMinus1_MatrixElements[16] =
+  { 0.999848, 0.0174524, 0, 0,   -0.0174524, 0.999848, 0, 0,   0, 0, 1, 0,   0, 0, 0, 1 };
+  if (!IsTransformMatrixEqualTo(mrmlScene,
+    iecLogic->GetTransformNodeBetween(vtkSlicerIECTransformLogic::PatientSupportRotation, vtkSlicerIECTransformLogic::FixedReference),
+    expectedPatientSupportRotationToFixedReferenceTransformMinus1_MatrixElements))
+  {
+    std:cerr << __LINE__ << ": PatientSupportRotationToFixedReference does not match baseline" << std::endl;
+    return EXIT_FAILURE;
+  }
+
+  double expectedBeamTransform_PatientSupportMinus1_MatrixElements[16] =
+  { 0.0174524, 0, -0.999848, 1000,   0, 1, 0, 200,   0.999848, 0, 0.0174524, 0,   0, 0, 0, 1 };

```

Comment:
Please add some indentation here, like two spaces, so it's obvious that this is the continuation of the previous line. All other places like this too please.